server/polar/order/tasks.py

```diff
@@ -134,3 +135,27 @@ async def order_invoice(order_id: uuid.UUID) -> None:
             raise OrderDoesNotExist(order_id)
 
         await order_service.generate_invoice(session, order)
+
+
+@actor(
+    actor_name="order.process_dunning",
+    cron_trigger=CronTrigger.from_crontab("0 * * * *"),
+    priority=TaskPriority.MEDIUM,
+)
+async def process_dunning() -> None:
+    """Process all orders that are due for dunning (payment retry)."""
+    async with AsyncSessionMaker() as session:
+        order_repository = OrderRepository.from_session(session)
+        due_orders = await order_repository.get_due_dunning_orders()
+
+    for order in due_orders:
+        async with AsyncSessionMaker() as session:
+            try:
+                await order_service.process_dunning_order(session, order)
+            except Exception as e:
+                sentry_sdk.capture_exception(e, extra={"order_id": str(order.id)})
```

Comment:
This is not necessary: Sentry automatically reports error logs.

We could even drop the `try..except` block to let it raise naturally.